inspire/modules/authors/config.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+## This file is part of Invenio.
+## Copyright (C) 2014 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+AUTHORS_IDENTIFIERS_TYPES = ['authorid', 'orcid', 'arxiv']
+
+INSPIRE_EMAIL_REGEX = "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
+INSPIRE_URL_REGEX = "^(?:http(?:s)?:\/\/)?(?:www\.)?(?:[\w-]*)\.\w{2,}$"
+
+INSPIRE_AFFILIATION_RANKS = ['senior', 'junior', 'staff', 'visitor', 'postdoc',
+                             'phd', 'masters', 'undergrad']
+
+# year - month (2 digits) -day (2 digits)
+INSPIRE_DATE_REGEX = "^[1|2][0-9]{3}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0|1])$"
+
+INSPIRE_AFFILIATION_STATUS = ['current']
+INSPIRE_EXPERIMENT_STATUS = ['current']
+
+INSPIRE_NAME_NUMERATIONS = "^Jr\.|Sr\.|III|IV|VI{0,3}$"
+INSPIRE_NAME_STATUSES = ['active', 'deceased', 'departed', 'retired']
+INSPIRE_NAME_TITLES = ['Sir']
+
+
+INSPIRE_YEAR_MIN = 1900
+INSPIRE_YEAR_MAX = 2050
+
+INSPIRE_ARXIV_CATEGORIES = ['acc-phys', 'astro-ph', 'atom-ph', 'chao-dyn',
+                            'climate', 'comp', 'cond-mat', 'genl-th', 'gr-qc',
+                            'hep-ex', 'hep-lat', 'hep-ph', 'hep-th', 'instr',
+                            'librarian', 'math', 'math-ph', 'med-phys', 'nlin',
+                            'nucl-ex', 'nucl-th', 'physics', 'plasma-phys',
+                            'q-bio', 'quant-ph', 'ssrl', 'other']
+
+INSPIRE_CATEGORIES_SOURCES = ['arxiv']
```

Comment:
Putting `AUTHORS` before each variable name would truly limit the variable _scope_. On the other hand, I think that some of them might be reusable in other modules and I didn't want to limit it so much.  For example `INSPIRE_ARXIV_CATEGORIES` may be used in future.